src/openMVG/robust_estimation/rand_sampling.hpp

```diff
@@ -35,10 +35,11 @@
 namespace openMVG {
 namespace robust{
 
+// rperrot : is this legal ? why using an anonymous namespace here ? 
 namespace
 {
   std::default_random_engine random_generator;
```

Comment:
Thread locale does not compile on OSX, just tested yet with current develop branch  : 

```
openMVG/src/openMVG/robust_estimation/rand_sampling.hpp:40:3: error: thread-local
      storage is not supported for the current target
  thread_local std::default_random_engine random_generator;
```